spatial_cell.cpp

```diff
@@ -74,18 +74,31 @@ namespace spatial_cell {
       }
    }
 
-   SpatialCell::SpatialCell(const SpatialCell& other):
-     sysBoundaryFlag(other.sysBoundaryFlag),
-     sysBoundaryLayer(other.sysBoundaryLayer),
-     velocity_block_with_content_list(other.velocity_block_with_content_list),
-     velocity_block_with_no_content_list(other.velocity_block_with_no_content_list),
-     initialized(other.initialized),
-     mpiTransferEnabled(other.mpiTransferEnabled),
-     derivativesBVOL(other.derivativesBVOL),
-     parameters(other.parameters),
-     populations(other.populations),
-     null_block_data(std::array<Realf,WID3> {}) {
-   }
+   // SpatialCell::SpatialCell(const SpatialCell& other):
```

Comment:
Instead of commenting them out (without a further comment), I would just outright delete them. We can always recover them from the git history, if need be.